lib/cisco_node_utils/cmd_ref/encapsulation.yaml

```diff
@@ -0,0 +1,22 @@
+# encapsulation profile
+# encapsulation profile feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k]
```

Comment:
You also need to evaluate `ios_xr` and exclude it if necessary. If you don't have access to an XR device then please ask for help from the larger team.